src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -517,6 +533,153 @@ p {
 	}
 }
 
+.receipt {
```

Comment:
@henryholtgeerts Can we create a separate file for receipt styling?

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/4615/commits/c907455f18f499073b462907f6663589c5319428